Classes/Domain/Search/ResultSet/ResultSetReconstitutionProcessor.php

```diff
@@ -124,7 +125,14 @@ protected function parseSortingIntoObjects(SearchResultSet $resultSet)
             $label = $sortingOptions['label'];
 
             $isResetOption = $field === 'relevance';
-            // @todo allow stdWrap on label
+
+            // Allow stdWrap on label:
+            $cObj = GeneralUtility::makeInstance(ContentObjectRenderer::class);
+            $labelHasSubConfiguration = is_array($sortingOptions['label.']);
+            if ($labelHasSubConfiguration) {
+                $label = $cObj->stdWrap($label, $sortingOptions['label.']);
+            }
```

Comment:
Don't initialize object, if it is not needed.
```suggestion
            $labelHasSubConfiguration = is_array($sortingOptions['label.']);
            if ($labelHasSubConfiguration) {
                $cObj = GeneralUtility::makeInstance(ContentObjectRenderer::class);
                $label = $cObj->stdWrap($label, $sortingOptions['label.']);
            }
```